### ISSUE TITLE
bug: iso info is no longer functional although still being offered

### ISSUE THREAD

**Issue by philclifford** [quickemu-project/quickemu#1088]
Helpful iso info is no longer functional although still being offered:

(This is broken on HEAD of main btw, not by this PR so I'll push this comment to a new issue)

```
# master and web_pipe
 ./quickget 
ERROR! You must specify an operating system.
 - Operating Systems: alma [...]zorin

        You can also use these arguments:
        Only show ISO download URL
            --show-iso-url / -s {distro} {release} [edition]
        Test if ISO is available
            --test-iso-url / -t {distro} {release} [edition]
        Open distro homepage
            --open-distro-homepage / -o {distro}
        Only download ISO
            --download-iso / -d {distro} {release} [edition]

# 4.9.1 -ish 
~/src/quickemu$ quickget --show-iso-url debian 12.5.0
https://cdimage.debian.org/debian-cd/12.5.0-live/amd64/iso-hybrid/debian-live-12.5.0-amd64-standard.iso

#HEAD of master and web_pipe
~/src/quickemu$ ./quickget --show-iso-url debian 12.5.0
ERROR! --show-iso-url is not a supported OS.

alma [...] zorin

```

same for long and short -s -t -d options 

_Originally posted by @philclifford in https://github.com/quickemu-project/quickemu/issues/1087#issuecomment-2067741852_

**Comment by zen0bit** [comment]
Sure is broken. Since I can't push all changes together, will will suffer till the new run script will be merged

**Comment by flexiondotorg** [comment]
Fixed via #999
